### DISCUSSION TITLE
Complete openfoam logs

### DISCUSSION THREAD

**Discussion by benoitpaillard** [mdolab/dafoam#680] [General]
Hi all,

Is there a way to get the whole log of openfoam runs, not just every 100 iterations ?

Thank you

**Comment by benoitpaillard** [comment]
Replying to my own question, I just found out this is controlled with daOptions : `"printInterval":10,`

**Comment by benoitpaillard** [comment]
Additional comment : often the deformed shape does not converge and it would be useful to be able to monitor convergence with pyfoam for instance, for which it would be great to have a specific external openfoam log file

**Comment by friedenhe** [comment]
Yeah, the reason we disabled the OpenFOAM's default log is that it is so long, and we can not afford to print every step in an optimization.
